app/src/main/java/com/powerhungers/fomezero/feature/registration/presentation/RegistrationViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package com.powerhungers.fomezero.feature.registration.presentation
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.powerhungers.fomezero.feature.registration.domain.usecase.RegistrationUseCase
+import kotlinx.coroutines.launch
+
+class RegistrationViewModel(private val registrationUseCase: RegistrationUseCase) : ViewModel() {
+
+    fun registration(
+        name: String, userType: String,
+        email: String, password: String
+    ) {viewModelScope.launch {
+        registrationUseCase
+    }
+
+    }
+
+
+}
```

Comment:
```suggestion
    fun registration(name: String, userType: String, email: String, password: String ) {
             viewModelScope.launch {
                   registrationUseCase
             }
    }
}
```
identação dificultando leitura do codigo